pro/src/pages/Offerers/Offerer/VenueV1/VenueEdition/VenueProvidersManager/AllocineProviderForm/AllocineProviderForm.tsx

```diff
@@ -80,104 +71,64 @@ const AllocineProviderForm = ({
-                  min="0"
-                  name="price"
-                  placeholder="Ex : 12€"
-                  step={0.01}
-                  required
-                  validate={required}
-                />
-              </div>
-              <div className="apf-quantity-section">
-                <label className="label-quantity" htmlFor="quantity">
-                  Nombre de places/séance
-                </label>
-                <NumberField
-                  isDecimal={false}
-                  min="0"
-                  name="quantity"
-                  className="quantity-field"
-                  placeholder="Illimité"
-                  onKeyPress={(e: React.KeyboardEvent<HTMLInputElement>) =>
-                    !/[0-9]|Backspace|Enter/.test(e.key) && e.preventDefault()
-                  }
-                  step={1}
-                />
-              </div>
-              <div className="apf-is-duo-section">
-                <CheckboxField
-                  id="apf-is-duo"
-                  label="Accepter les réservations DUO"
-                  name="isDuo"
-                />
-                <span
-                  className="apf-tooltip"
-                  data-tooltip-place="bottom"
-                  data-tooltip-html="<p>En activant cette option, vous permettez au bénéficiaire du pass Culture de venir accompagné. La seconde place sera délivrée au même tarif que la première, quel que soit l’accompagnateur.</p>"
-                  data-tooltip-type="info"
-                  data-tooltip-id="tooltip-duo"
-                >
-                  <SvgIcon src={strokeInfoIcon} alt="" className="info-icon" />
-                </span>
-                <Tooltip
-                  className="type-info flex-center items-center"
-                  delayHide={500}
-                  id="tooltip-duo"
-                />
-              </div>
+  const formik = useFormik({
+    initialValues,
+    onSubmit: handleSubmit,
+    validationSchema,
+  })
 
+  return (
+    <FormikProvider value={formik}>
+      <FormLayout>
+        {!isLoading && (
+          <>
+            <FormLayout.Row
+              sideComponent={
+                isCreatedEntity ? (
+                  <InfoBox>Prix auquel la place de cinéma sera vendue.</InfoBox>
+                ) : (
+                  <></>
+                )
+              }
+            >
+              <TextInput
+                name="price"
+                type="number"
+                label="Prix de vente/place"
+                min="0"
+                placeholder="Ex : 12€"
+                step={0.01}
+                required
+              />
+            </FormLayout.Row>
+            <FormLayout.Row>
+              <TextInput
+                type="number"
+                label="Nombre de places/séance"
+                min="0"
+                name="quantity"
+                placeholder="Illimité"
+                step={1}
```

Comment:
Pour éviter à l'utilisateur de rentrer un nombre à virgule
```suggestion
                step={1}
                hasDecimal={false}
```